src/modules/colorPicker/ColorPickerUI/Views/ColorEditorView.xaml

```diff
@@ -1,212 +1,294 @@
-                    ToolTipService.ToolTip="{x:Static p:Resources.Open_settings}"
-                    AutomationProperties.Name="{x:Static p:Resources.Open_settings}" />
+                                        </MultiTrigger>
+                                        <!--  PointerOverSelected  -->
+                                        <MultiTrigger>
+                                            <MultiTrigger.Conditions>
+                                                <Condition Property="IsMouseOver" Value="True" />
+                                                <Condition Property="IsSelected" Value="True" />
+                                            </MultiTrigger.Conditions>
+                                            <Setter TargetName="SelectionStroke" Property="Opacity" Value="1" />
+                                            <Setter TargetName="RootGrid" Property="Opacity" Value="0.8" />
+                                        </MultiTrigger>
+                                        <!--  Disabled  -->
+                                        <Trigger Property="IsEnabled" Value="False">
+                                            <Setter TargetName="RootGrid" Property="Opacity" Value="{DynamicResource ListViewItemDisabledThemeOpacity}" />
+                                        </Trigger>
+                                    </ControlTemplate.Triggers>
+                                </ControlTemplate>
+                            </Setter.Value>
+                        </Setter>
+                    </Style>
+                </ListView.ItemContainerStyle>
+            </ListView>
 
-            <Button Width="64"
-                    Height="32"
-                    TabIndex="0"
-                    Command="{Binding OpenColorPickerCommand}"
-                    Background="Transparent"
-                    FontFamily="Segoe MDL2 Assets"
-                    ui:FocusVisualHelper.FocusVisualMargin="-1"
-                    ToolTipService.ToolTip="{x:Static p:Resources.Pick_color}"
-                    AutomationProperties.Name="{x:Static p:Resources.Pick_color}">
+            <Button
+                Grid.Column="2"
+                Width="36"
+                Height="36"
+                Padding="0"
+                HorizontalAlignment="Right"
+                VerticalAlignment="Top"
+                AutomationProperties.Name="{x:Static p:Resources.Open_settings}"
+                Background="Transparent"
+                BorderBrush="Transparent"
+                Command="{Binding OpenSettingsCommand}"
+                Style="{StaticResource SubtleButtonStyle}"
+                TabIndex="2"
+                ToolTipService.ToolTip="{x:Static p:Resources.Open_settings}">
                 <Button.Content>
-                    <TextBlock Text="&#xEF3C;" AutomationProperties.Name="{x:Static p:Resources.Pick_color}" />
+                    <ui:SymbolIcon FontSize="18" Symbol="Settings20" />
```

Comment:
20 and 20 would give a nice sharp icon

Reply:
It's getting too big then :(, changed it to 20 for the icon though and looks sharp on my screens.